azure-pipelines.yml

```diff
@@ -77,28 +78,35 @@ stages:
        - script: cmake -S . -B ./build/ -DUSE_QT6:BOOL=FALSE
          displayName: "CMake: Create Project (Linux)"
          condition: and(succeeded(), eq( variables['Agent.OS'], 'Linux'))
-       - script: cmake --build ./build --verbose
-         displayName: "CMake: Compile and Link"
+       - script: cmake --build ./build --config Debug --verbose
+         displayName: "CMake: Compile and Link (Debug)"
+         condition: succeeded()
+       - script: cmake --build ./build --config Release --verbose
+         displayName: "CMake: Compile and Link (Release)"
          condition: succeeded()
        - script: ./build/test/TEST-$(project_name)
          displayName: "Run Tests on Linux / Mac OS"
          condition: and(succeeded(), ne( variables['Agent.OS'], 'Windows_NT'))
-       - script: .\build\test\Debug\TEST-$(project_name).exe
+       - script: tree /F .\build
+         displayName: "View tree of 'build\' on Windows"
```

Comment:
you don't need escaping here...

Reply:
same below

Reply:
The trailing (back)slash is generally used to mark a directory.